### PR TITLE
Add source code for ExtractPolyLinesFromPolyData

### PR DESCRIPTION
**Pull request by jaswantp**
Ported from Cxx.

### CODE REVIEW

**Comment by jaswantp** [comment]
added image

File: src/Python/PolyData/ExtractPolyLinesFromPolyData.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    print("There are {0} lines in the polydata".format(numberOfLines))
+
+    indices = vtk.vtkIdList()
+    lineCount = 0
+
+    while cells.GetNextCell(indices):
+        print("Line {0}:".format(lineCount))
+        for i in range(indices.GetNumberOfIds()):
+            point = points.GetPoint(indices.GetId(i))
+            print("\t({0} ,{1}, {2})".format(point[0], point[1], point[2]))
```
Comment (ajpmaclean) [inline src/Python/PolyData/ExtractPolyLinesFromPolyData.py]: I would suggest formatting to at most six decimals. Something like this should do it:

``` python
            print("\t({0:0.6f}, {1:0.6f}, {2:0.6f})".format(point[0], point[1], point[2]))

```

Comment (jaswantp) [reply src/Python/PolyData/ExtractPolyLinesFromPolyData.py]: Yes, that is better.

**Comment by ajpmaclean** [comment]
THankyou for your contribution.

**Pull request merged**